pmm/v3/pmm3-package-tests-matrix.groovy

```diff
@@ -2,17 +2,121 @@ library changelog: false, identifier: 'lib@master', retriever: modernSCM([
     $class: 'GitSCMSource',
     remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
 ]) _
-void runPackageTestingJob(String GIT_BRANCH, DOCKER_VERSION, PMM_VERSION, TESTS, METRICS_MODE, INSTALL_REPO) {
-    upgradeJob = build job: 'pmm-package-testing', parameters: [
+
+void run_package_tests(String GIT_BRANCH, String PLAYBOOK, String INSTALL_REPO)
+{
+    deleteDir()
+    git poll: false, branch: GIT_BRANCH, url: 'https://github.com/Percona-QA/package-testing'
+    sh """
+        export install_repo=${INSTALL_REPO}
+        export TARBALL_LINK=${TARBALL}
+        ls playbooks
+        ansible-playbook \
+        -vvv \
+        --connection=local \
+        --inventory 127.0.0.1, \
+        --limit 127.0.0.1 playbooks/${PLAYBOOK}.yml
+    """
```

Comment:
```suggestion
    sh '''
        export install_repo=${INSTALL_REPO}
        export TARBALL_LINK=${TARBALL}
        ls playbooks
        ansible-playbook \
        -vvv \
        --connection=local \
        --inventory 127.0.0.1, \
        --limit 127.0.0.1 playbooks/${PLAYBOOK}.yml
    '''
```

Reply:
I suggest we default to single quotes for `sh`, since Jenkins has a security issue with double quotes.

Reply:
Changes done.